pages/Profile.js

```diff
@@ -72,24 +78,50 @@ const Profile = () => {
 
         tempPosseData.push(header);
         tempPosseData.push(buttons);
-        res.data.posses.forEach((f, index) => {
-          let tempPosse = {};
-          tempPosse.id = (index + 3).toString();
-          tempPosse.name = f;
-          tempPosse.imagePath = 'https://picsum.photos/200';
-          tempPosse.type = 'posse';
-
-          tempPosseData.push(tempPosse);
-        });
+
+        if (res.data.posses) {
+          res.data.posses.forEach((f, index) => {
+            let tempPosse = {};
+            tempPosse.id = (index + 3).toString();
```

Comment:
I was doing this too in my branch a while ago, but it is bad practice. We should id these flat list entities by posseID instead of index. This was the root of my feed update problems.